pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -2420,10 +2420,7 @@ func TestPipelineTasksExecutionStatus(t *testing.T) {
 		}},
 		expectedError: *apis.ErrGeneric("").Also(&apis.FieldError{
 			Message: `invalid value: pipeline tasks can not refer to execution status of any other pipeline task or aggregate status of tasks`,
-			Paths:   []string{"tasks[0].params[bar-status].value"},
```

Comment:
Did we change anything to avoid reporting these errors?

Reply:
removed the `when expressions in` prefix in the message because it's redundant - we already use the field keys and indices to show where the disallowed field is - that's why it collapses into one error with multiple paths